src/india_api/internal/config/env.py

```diff
@@ -56,7 +56,7 @@ def __init__(self) -> None:
 class Config(EnvParser):
     """Config for the application."""
 
-    SOURCE: str = "indiadb"
+    SOURCE: str = "dummydb"
```

Comment:
Was this change intentionally left in? Not sure if this is 100% being set in the env but probs safer to have the default as the real/prod value in this case..?

Reply:
Yea my mistake, I'll change that bk

Reply:
This should be tidied up now, but I'm on my phone so I can't quite see